hat/app/org/hatdex/hat/modules/SHEModule.scala

```diff
@@ -63,7 +63,7 @@ class SHEModule extends AbstractModule with ScalaModule with AkkaGuiceSupport {
             config.getBoolean("experimental")
           )
         }
-      }
+      }.toSeq
```

Comment:
Just curious, why this is needed now and not before? How did it compile before? 😮

Reply:
Scala 2.12 -> 2.13
in 2.12 `scala.Seq` can be mutable
in 2.13 it is an alias for `scala.collection.immutable.Seq[A]` i.e. is immutable (which is the way it should always have been in a functional language)

Here the sequence is built from an array and is mutable, `.toSeq` gives the immutable version

Reply:
since this is the return type of the method